core/src/test/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -271,6 +277,27 @@
      */
     public static final String TESTS_ENABLE_MOCK_MODULES = "tests.enable_mock_modules";
 
+    /*
+     * These are a few backwards compatibility test constants that are declared
+     * here so they can be used in both ESBackcompatTestCase and
+     * ReproduceInfoPrinter.
+     */
+    /**
+     * Key used to set the path for the elasticsearch executable used to run backwards compatibility tests from
+     * via the commandline -D{@value #TESTS_BACKWARDS_COMPATIBILITY}
+     */
+    public static final String TESTS_BACKWARDS_COMPATIBILITY = "tests.bwc";
```

Comment:
Yeah - I agree - I think this is nasty. I did it without a whole lot of thought trying to get the move working.